pslab/instrument/oscilloscope.py

```diff
@@ -132,6 +132,10 @@ def capture(
 
         self._check_args(channels, samples, timegap)
         timegap = int(timegap * 8) / 8
+        
+        for channel in ("CH1", "CH2"):
+            self._set_gain(channel, gain)
```

Comment:
The gain value is stored in `AnalogInput` instances, which are in turn stored in the `_channels` attribute of the `Oscilloscope` class. You can access the stored gain values like this:
```python
gain = self._channels[channel].gain
```
Where `channel` is the name of the channel in question (CH1 or CH2) as a string.